### PR TITLE
Update update-daemon-set.md

### PR DESCRIPTION
**Pull request by guzj11**
sync with english version.

### CODE REVIEW

File: content/zh/docs/tasks/manage-daemon/update-daemon-set.md [hunk 1]
```diff
@@ -234,6 +234,17 @@ daemonset "fluentd-elasticsearch" successfully rolled out
 
 ### DaemonSet 滚动更新卡住
 
+<!--
+Sometimes, a DaemonSet rolling update may be stuck. Here are some possible
+causes:
+
+#### Some nodes run out of resources
+-->
+有时，DaemonSet 滚动更新可能卡住，以下是一些可能的愿意：
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-daemon/update-daemon-set.md:243]: 原因

File: content/zh/docs/tasks/manage-daemon/update-daemon-set.md [hunk 2]
```diff
@@ -234,6 +234,17 @@ daemonset "fluentd-elasticsearch" successfully rolled out
...
+
+
+### 一些节点可用资源耗尽
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-daemon/update-daemon-set.md:246]:
```suggestion
#### 一些节点可用资源耗尽
```

Comment (guzj11) [reply content/zh/docs/tasks/manage-daemon/update-daemon-set.md]: Thank you for suggestion
suggestion applied

**Pull request merged**